pkg/loadbalancer/experimental/testdata/clusterip.txtar

```diff
@@ -30,8 +30,8 @@ Address               Type        ServiceName   PortName   Backends
 10.96.50.104:80/TCP   ClusterIP   test/echo     http       10.244.1.1:80/TCP (active)   Done
 
 -- backends.table --
-Address             State    Instances          NodeName          ZoneID
-10.244.1.1:80/TCP   active   test/echo (http)   nodeport-worker   0
+Address             State    Instances          Shadows        NodeName          ZoneID
+10.244.1.1:80/TCP   active   test/echo (http)   test/echo []   nodeport-worker   0
```

Comment:
Why are we seeing `test/echo []`? Perhaps we should omit when there's nothing.

Reply:
> Why are we seeing test/echo []? Perhaps we should omit when there's nothing.

You commented above that you don't want to see a preferred instance in both Instances and Shadows, which is why there's nothing. Since for now we don't have support for different sources in script test, Shadows will always be empty. I can omit it altogether, but I wanted the shadow feature to be somehow reflected in script tests, which is the reason for this 2-line change in `clusterip.txtar`. Can we leave it as it is?

Reply:
Resolving in the light of Jussi's approval below.